---
layout: archive
title: "Research Highlights"
permalink: /research/
author_profile: true
---


<h3 class="archive__item-title" itemprop="headline">
      
1. Cross species resemblance analysis of genomic response. 
      
  </h3>

<p></p>

  <div class="figure">
    
    	<img src="/files/figures/CAMO.jpg" alt="">
    
  </div>


<ul style="list-style-type:disc">

<li> Zong W, Rahman T, Zhu L, Zeng X, Zhang Y, Zou J, Liu S, Li JJ, <b> Ma T*</b> and Tseng GC*. (2021+). Congruence evaluation for model organisms in transcriptomic response. Ready to submit. </li>
<li> (Motivating PNAS paper #1) Seok et al. (2013). Genomic responses in mouse models poorly mimic human inflammatory diseases. <i>Proceedings of the National Academy of Sciences </i>, 110(9), 3507-3512.  </li> 
<li> (Motivating PNAS paper #2) Takao, K., & Miyakawa, T. (2015). Genomic responses in mouse models greatly mimic human inflammatory diseases. <i>Proceedings of the National Academy of Sciences </i>, 112(4), 1167-1172. </li> 
</ul>

<p></p><p></p>


<h3 class="archive__item-title" itemprop="headline">
      
2. Post-GWAS statistical genetic methods (e.g. fine mapping, pleiotropy, causal inference).
      
  </h3>

<p></p>

  <div class="figure">
    
    	<img src="/files/figures/FM_LD.jpg" alt="">
    
  </div>

<p></p>

  <div class="figure">
    
    	<img src="/files/figures/Pleiotropy.jpg" alt="">
    
  </div>


<ul style="list-style-type:disc">

<li> <u> Ye Z^</u>, Mo C^, Hatch K, Liu S, Gao S, Hong E, Kochunov P*, Chen S* and <b> Ma T*</b>. (2021). White matter integrity and nicotine dependence in smokers: evaluating vertical and horizontal pleiotropy. <i>bioRxiv</i>. <a href="https://www.biorxiv.org/content/10.1101/2020.12.09.417899v1"> [Link]</a> </li>
<li> Mo C^, <u> Ye Z^ </u>, Hatch K, Zhang Y, Wu Q, Liu S and Kochunov P, <b> Ma T* </b> and Chen S*. (2021). Genetic Fine-mapping with Dense Linkage Disequilibrium Blocks: genetics of nicotine dependence. <i>bioRxiv</i>. <a href="https://www.biorxiv.org/content/10.1101/2020.12.10.420216v1"> [Link]</a> </li> 
</ul>

<p></p><p></p>

<h3 class="archive__item-title" itemprop="headline">

3. Variable screening, selection and categorization with multiple studies (Pan-Cancer application)
      
  </h3>

<p></p>

  <div class="figure">
    
    	<img src="/files/figures/TSA_SIS.jpg" alt="">
    
  </div>

<p></p>

  <div class="figure">
    
    	<img src="/files/figures/BCMC.jpg" alt="">
    
  </div>


<ul style="list-style-type:disc">

<li> <u> Ye Z^</u>, <u> Ke H^</u>, Chen S, ..., Milton D and <b>Ma T* </b>. (2021+). Biomarker categorization in transcriptomic meta-analysis by concordant patterns with application to Pan-cancer studies. Under Revision in <i> Frontiers in Genetics </i>. <a href="https://github.com/kehongjie/BCMC">[software]</a> </li> 

<li> Saegusa T, <u> Zhao Z </u>, <u> Ke H </u>, <u> Ye Z </u>, Xu Z, Chen S and  <b> Ma T* </b>. (2021). Detecting survival-associated biomarkers from heterogeneous populations. <i>Scientific Reports </i>, Accepted. <a href="files/preprints/CoxTOTEM.pdf">[pdf]</a> <a href="https://github.com/kehongjie/CoxTOTEM">[software]</a> </li>
<li> <b> Ma T </b>, Ren Z and Tseng GC. (2020). Variable screening with multiple studies. <i>Statistica Sinica </i>, 30(2): 925–953.<a href="files/preprints/TSA-SIS.pdf">[pdf]</a> </li>

</ul>

<p></p><p></p>


<h3 class="archive__item-title" itemprop="headline">
   
4. Meta-analysis of multiple transcriptomic studies 
      
  </h3>

<p></p>


  <div class="figure">
    
    	<img src="/files/figures/MetaOmics.jpg" alt="">
    
  </div>

<p></p>

  <div class="figure">
    
    	<img src="/files/figures/BayesMetaSeqCBM.jpg" alt="">
    
  </div>


<ul style="list-style-type:disc">

<li> <b> Ma T^</b>, Huo Z^, Kuo A^, Zhu L, ..., Song C and Tseng GC. (2019). MetaOmics - Comprehensive Analysis Pipeline and Web-based Software Suite for Transcriptomic Meta-Analysis. <i>Bioinformatics </i>.  <a href="https://www.ncbi.nlm.nih.gov/pubmed/30304367">PMID: 30304367</a>. <a href="files/preprints/MetaOmics.pdf">[pdf]</a> <a href="https://github.com/metaOmics/metaOmics">[software]</a> </li>
<li> <b> Ma T </b>, Liang F. and Tseng GC. (2017). Biomarker detection and categorization in ribonucleic acid sequencing meta-analysis using Bayesian hierarchical model. <i>Journal of the Royal Statistical Society: Series C</i>. 66(4): 847-867. <a href="https://matianzhou.github.io/files/preprints/BayesMetaSeq.pdf">[pdf]</a> </li>
<li> <b> Ma T </b>, Liang F, Oesterreich S and Tseng GC. (2017). A joint Bayesian modeling for integrating microarray and RNA-seq transcriptomic data. <i>Journal of Computational Biology</i>. 24(7): 647-662. <a href="https://matianzhou.github.io/files/preprints/CBM.pdf">[pdf]</a> </li>
</ul>

<p></p><p></p>


<h3 class="archive__item-title" itemprop="headline">
      

5. Multi-omics data integration 

  </h3>

<p></p>

  <div class="figure">
    
    	<img src="/files/figures/MultiOmics.jpg" alt="">
    
  </div>


<ul style="list-style-type:disc">

<li> Zhu L, Huo Z, <b> Ma T</b>, Tseng GC. (2019). Bayesian indicator variable selection to incorporate hierarchical overlapping group structure in multi-omics applications. <i> Annals of Applied Statistics </i>, 13(4): 2611-2636. (a preliminary version won the ENAR distinguished student paper award). </li>
<li> Huo Z, Zhu L, <b> Ma T</b>, Liu H, Han S, Liao D, Zhao J and Tseng GC. (2019). Two-way Horizontal and Vertical Omics Integration for Disease Subtype Discovery. <i> Statistics in Biosciences </i>, 1-22. </li>
<li> Fang Z, <b>Ma T</b>, Tang G, Zhu L, Yan Q, Wang T, Celedón JC, Chen W and Tseng GC.(2018). Bayesian integrative model for multi-omics data with missingness. <i>Bioinformatics </i>, 34(22): 3801-3808. <a href="https://www.ncbi.nlm.nih.gov/pubmed/30184058">PMID: 30184058</a>. </li>
<li> Liao S, Hartmaier RJ, McGuire KP, Puhalla SL, Luthra S, Chandran UR, <b>Ma T</b>, Bhargava R, Davidson NE, Benz S, Lee AV, Tseng GC and Oesterreich S. (2015). The molecular landscape of premenopausal breast cancer. <i>Breast Cancer Research</i>, 17:104. doi: 10.1186/s13058-015-0618-8. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26251034">PMID: 26251034</a>. (discussed in an interview; <a href="http://www.nature.com/nature/journal/v527/n7578_supp/full/527S108a.html"><i>Nature</i>, 527: S108-109</a>) </li>
</ul>

